---
title: "s"
output: html_document
date: "2024-11-14"
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

```{r}
library(Seurat)
```


```{r}
library(dplyr)
library(ggplot2)
library(SeuratObject)
library(Signac)
library(MAST)

# Load the Seurat object and metadata
load("C:/Users/sanet/Downloads/multiomics_final_objects_metadata/multiomics_final_objects_metadata/seurat_data_brain_cc_2022-12-22.RData")


# Verify that metadata has been added and the "cell_flight_status" column is available
head(seurat_multi@meta.data)

# Set cell identities for DGE analysis based on "cell_flight_status"
Idents(seurat_multi) <- seurat_multi@meta.data$cell_flight_status

seurat_multi_VF <- FindVariableFeatures(seurat_multi, selection.method = "vst", nfeatures = 2000)
top10 <- head(VariableFeatures(seurat_multi_VF), 10)
plot1 <- VariableFeaturePlot(seurat_multi)
plot2 <- LabelPoints(plot = plot1, points = top10, repel = TRUE)
plot2
plot1 + plot2

DefaultAssay(seurat_multi) <- "RNA"
```

```{r}

```

```{r}
# Perform differential expression analysis
DefaultAssay(seurat_multi) <- "RNA"
markers <- FindMarkers(
  object = seurat_multi,
  ident.1 = "flight",      # Group 1
  ident.2 = "ground",      # Group 2
  test.use = "wilcox",     # Wilcoxon rank-sum test
  min.pct = 0.30,          # Only include genes expressed in at least 25% of cells
  logfc.threshold = 0.50, assay = "RNA"   # Minimum log2 fold change of 0.25
)

```
```{r}
expression_data <- seurat_multi@assays$RNA@data[rownames(markers), ]

```

```{r}
library(pheatmap)

# Generate a heatmap with annotation for flight status
pheatmap(
  expression_data,
  scale = "row",  # Scale expression values across genes for better contrast
  cluster_rows = TRUE,  # Cluster genes
  cluster_cols = TRUE,  # Cluster cells
  show_rownames = TRUE,  # Show gene names
  show_colnames = FALSE,  # Optionally hide cell names
  group.by = "cell_flight_status",,  # Add metadata for cells
  color = colorRampPalette(c("blue", "white", "red"))(50)  # Custom color scale
)

```




```{r}
library(EnhancedVolcano)
library(dplyr)
#1 Volcano Plot
markers3=as.data.frame(markers)
markers3=mutate(markers, sig=ifelse(markers$p_val_adj<0.05,"Sig","Not Sig"))
markers3[which(abs(markers$avg_log2FC)<0.5), 'sig'] <- "Not Sig"
log2FoldChange_graph=ggplot(markers3, aes(avg_log2FC, -log10(p_val_adj)))+geom_point(aes(col=sig))+scale_color_manual(values=c("red","black"))
log2FoldChange_graph

hist(markers3$avg_log2FC, main = "Distribution of avg_log2FC", xlab = "avg_log2FC")
hist(markers3$p_val_adj, main = "Distribution of p_val_adj", xlab = "p_val_adj")


```


```{r}
# View the top markers3
df <- read.csv("C:/Users/sanet/OneDrive/Documenti/Trento/UNI/Data Mining - Progetto/DEG2.csv")
head(markers)
options(digits = 15)
markers2 <- subset(markers , markers$p_val_adj<0.05)
#write.csv(as.data.frame(markers2), "DEG2.csv")
positive_deg <- subset(df , df$avg_log2FC > 0)
negative_deg <- subset(df , df$avg_log2FC<0)
write.csv(as.data.frame(positive_deg), "PDEG.csv")
write.csv(as.data.frame(negative_deg), "NDEG.csv")
```


```{r}
```


```{r}
library(readxl)
df_alz <- read_excel("C:/Users/sanet/OneDrive/Documenti/Trento/UNI/Data Mining - Progetto/geneTabReportALZ.xlsx")
library(ggVennDiagram)
DEGs <- df$X
AlzheimerGenes <-df_alz$`Mouse Homologs`
sets_list <- list(DEGs = DEGs, AlzGenes = AlzheimerGenes)
ggVennDiagram(sets_list) 
common_genes1 <- Reduce(intersect, sets_list)
common_genes1
```
```{r}
df_GRIN2B <- c("ABAT", "ADD2", "ANKS1B", "ARHGEF12", "ARPP21", "ASTN1", "B3GAT1", 
           "C1QTNF4", "C1QTNF4", "C4orf44", "C6orf138", "C8orf34", "CACNA1G", 
           "CAMK2A", "CAMK2N1", "CELF5", "CELF5", "CSMD1", "DCLK1", "DLG2", 
           "DLG3", "ELMO1", "FAM5C", "FMNL1", "FXYD7", "GABBR2", "GNAO1", 
           "GPM6A", "GPM6A", "GPM6A", "GRIN2B", "GRIN2B", "GRM7", "KCNC1", 
           "KCNC2", "KCNF1", "KCNJ6", "KCNV1", "KIF3C", "LINC00340", "LPHN2", 
           "LRFN2", "LRRC7", "MAPK8IP2", "MATR3", "MEF2C", "MEF2C", "MGAT5B", 
           "MIR137HG", "MPPED1", "NAV1", "NAV1", "NAV1", "NRXN1", "NRXN1", 
           "NRXN2", "OLFM3", "OPRL1", "PALM2", "PGM2L1", "PPIE", "PTPRD", 
           "PTPRD", "RBFOX1", "RBFOX1", "RIMBP2", "RTN4RL1", "SCN3B", "SCN3B", 
           "SHANK2", "SHANK2", "SHANK2", "SORCS1", "SYT5", "TNRC18", "TRAK1", 
           "TSPAN14", "TSPAN14", "TTC9B", "XKR4", "XKR4")
df_GRIN2B <- unique(df_GRIN2B)
library(readxl)
degs <- read_excel("C:/Users/sanet/OneDrive/Documenti/Trento/UNI/Data Mining - Progetto/DEGS_H.xlsx", col_names = FALSE)
colnames(degs)[colnames(degs) == "...1"] <- "genes"
degs <- degs$genes
sets_list <- list(DEGs = degs, GRIN2B = df_GRIN2B)
ggVennDiagram(sets_list) 
common_genes_GRIN2B <- Reduce(intersect, sets_list)
common_genes_GRIN2B
```


```{r}
df_KCNMA1 <- c("VIM", "MKX", "KCNMA1", "KCNMA1", "KCNMA1", "KCNMA1", "KCNMA1", 
           "KCNMA1", "KCNMA1", "KCNMA1", "KCNMA1", "KCNMA1", "KCNMA1", 
           "KCNJ5", "CTSD", "NAV2", "QSER1", "DRAP1", "DRAP1", "DRAP1", 
           "DRAP1", "NLRP10", "CCDC92", "NCOR2", "WNT5B", "ARNTL2", 
           "ANKRD52", "CD163L1", "RXFP2", "FRMD6", "NRXN3", "CALM3", 
           "C15orf52", "NEDD4", "SLC9A3R2", "HS3ST2", "KRTAP1-5", "STAT3", 
           "TRIM47", "BAIAP2", "FHOD3", "ATP8B1", "CCBE1", "CCBE1", "CCBE1", 
           "NFIC", "AXL", "AXL", "AXL", "AXL", "CEACAMP10", "NGF", "VPS13D", 
           "TNFSF18", "RASAL2", "LYPLAL1", "C1QB", "C1QB", "GNG4", "KLF17", 
           "LHX8", "PMEPA1", "PLCB4", "PLCB4", "PLCB4", "PLCB4", "PLCB4", 
           "LINC00313", "TRPM2", "KIAA1644", "KIAA1644", "HPCAL1", "IGFBP5", 
           "IRS1", "HEATR7B1", "HEATR7B1", "RNPEPL1", "FAM176A", "PHLDB2", 
           "BOC", "TPRA1", "ZBTB38", "ZBTB38", "ZBTB38", "EIF4G1", "DLG1", 
           "RARB", "OXTR", "PALLD", "VEGFC", "AMTN", "FGF5", "CCDC99", "FST", 
           "FST", "RGMB", "RGMB", "RGMB", "RGMB", "RGMB", "RGMB", "RGMB", 
           "FOXF2", "NT5E", "ZYX", "NKX3-1", "DOCK5", "NRG1", "NRG1", 
           "CEBPD", "BNC2", "BARX1", "STS", "SHROOM2")


df_KCNMA1 <- unique(df_KCNMA1)
sets_list <- list(DEGs = degs, KCNMA1 = df_KCNMA1)
ggVennDiagram(sets_list) 
common_genes_KCNMA1 <- Reduce(intersect, sets_list)
common_genes_KCNMA1

```
```{r}
df_PLBC1 <- c("ADCY9", "ADD1", "ARHGAP32", "ARHGAP32", "ATP2B1", "C10orf112", 
           "C9orf93", "CACNB4", "CAMK4", "CES4A", "CHRM3", "CNNM1", "CNST", 
           "CORO2B", "CSGALNACT1", "DPP6", "EPHB6", "FRY", "FRY", "FRY", 
           "GABARAPL1", "GABRA2", "GNAO1", "GPR158", "GPR158", "GUCY1A3", 
           "GUCY1A3", "GUCY1A3", "GUCY1B3", "HLF", "KCNQ5", "KIAA1377", 
           "LDB2", "LDB2", "NEU3", "NOVA1", "NRGN", "OPCML", "PDE2A", 
           "PDE8B", "PLCB1", "PLCB1", "PPP1R9A", "REPS2", "RGS7BP", "RGS7BP", 
           "RPS6KC1", "SMARCA2", "ST8SIA3", "SUGP2", "SULT1A1", "SYNGAP1", 
           "TMX4", "TRANK1", "UBR3", "ZNF385B", "ZNF804A")
df_PLBC1 <- unique(df_PLBC1)
sets_list <- list(DEGs = degs, PLBC1 = df_PLBC1)
ggVennDiagram(sets_list)
common_genes_PLBC1 <- Reduce(intersect, sets_list)
common_genes_PLBC1

```
```{r}
df_NRTK2 <- c("TRIL", "THRA", "THRA", "SUGT1P3", "SRI", "SPTBN1", "SPON1", 
           "SLC6A1", "SLC22A17", "SLC1A2", "SLC1A2", "SLC1A2", "SCD5", 
           "RTN1", "RPS6KA2", "RAPGEF3", "RAPGEF3", "PSD2", "PREX2", 
           "PNMAL2", "PHYHIPL", "PHYHIPL", "PDZD2", "PCDH17", "PAPLN", 
           "NTSR2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", 
           "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", 
           "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NTRK2", "NKAIN3", 
           "NDRG2", "NDRG2", "NAT8L", "MAPK10", "KCNN3", "KCNA2", "KATNAL2", 
           "IQCA1", "HSPA12A", "GRIK4", "GPR75", "GPR37L1", "GPM6A", 
           "GABBR1", "FMN2", "FAM47E", "FAM13C", "ENHO", "CERS1", "ATP2B3", 
           "ATP1A2", "ARNT2", "ARHGEF26-AS1", "ARHGEF26-AS1", 
           "ALDH1L1-AS2", "AIFM3", "ADCYAP1R1", "ADCYAP1R1", "ADCY2")
df_NRTK2 <- unique(df_NRTK2)
sets_list <- list(DEGs = degs, NRTK2 = df_NRTK2)
ggVennDiagram(sets_list)
common_genes_NRTK2 <- Reduce(intersect, sets_list)
common_genes_NRTK2
```


```{r}
df_APBB2 <- c("DOCK1", "PHLDB1", "PHLDB1", "NUAK1", "NCOR2", "NCOR2", "SPATS2", 
           "SPATS2", "SPATS2", "SRGAP1", "PTPRB", "SYT1", "STARD13", "DZIP1", 
           "CDC42BPB", "MIPOL1", "MIPOL1", "MIPOL1", "MAP4K5", "PTPN21", 
           "PPP4R4", "TJP1", "UACA", "CHD9", "DYNC1LI2", "CUEDC1", "ZBTB4", 
           "CABYR", "YES1", "CTTNBP2NL", "PDE4DIP", "OTUD7B", "RASAL2", 
           "TGFB2", "TGFB2", "TGFB2", "TGFB2", "TGFB2", "CDC42BPA", "KIAA0754", 
           "GNG12", "GNG12", "ERRFI1", "EVI5", "EVI5", "BCAR3", "CNN3", 
           "SNX7", "BFSP1", "RIN2", "RIN2", "RIN2", "APP", "RBFOX2", "CDC42EP1", 
           "CDC42EP1", "NPAS2", "PSD4", "TANC1", "NCKAP1", "ABI2", "RAPH1", 
           "RAPH1", "RAPH1", "USP40", "MTA3", "ANTXR1", "ASAP2", "ASAP2", 
           "ASAP2", "ZBTB20", "ZBTB20", "ATP2C1", "AMOTL2", "TRPC1", "PLOD2", 
           "PLOD2", "PLOD2", "PLSCR4", "ARHGEF26", "SCHIP1", "SCHIP1", 
           "SCHIP1", "EIF5A2", "DLG1", "RARB", "RARB", "RBMS3", "RBMS3", 
           "OSBPL10", "FBXL2", "MAP4", "SMAD1", "FAM200B", "SH3RF1", "WWC2", 
           "FAT1", "KLHL5", "APBB2", "APBB2", "APBB2", "APBB2", "APBB2", 
           "APBB2", "UCHL1", "SLC4A4", "FRAS1", "AFAP1", "AFAP1", "MCC", 
           "NDST1", "RANBP17", "EDIL3", "EDIL3", "EDIL3", "EDIL3", "PTPRK", 
           "PHACTR1", "ABT1", "LINC00472", "WASL", "PDGFA", "CDK14", "DLC1", 
           "PTK2", "NRBP2", "ARHGEF10", "PALM2-AKAP2", "KLF9", "SPIN1", 
           "LDOC1", "PRRG1", "UPRT")
df_APBB2 <- unique(df_APBB2)
sets_list <- list(DEGs = degs, APBB2 = df_APBB2)
ggVennDiagram(sets_list)
common_genes_APBB2 <- Reduce(intersect, sets_list)
common_genes_APBB2

```
```{r}
df_GSK3B <- c("C10orf46", "EXOSC1", "RPUSD4", "TP53I11", "MRPL16", "ARL2", 
           "PRDM4", "RPS26", "BAZ2A", "PLXNC1", "C13orf27", "CHD8", "DCAF5", 
           "PCNX", "BTBD7", "IGF1R", "C16orf72", "TOB1", "CUEDC1", "VEZF1", 
           "RPS6KB1", "USP32", "USP32", "TEX2", "GNA13", "BPTF", "ST6GALNAC2", 
           "ST6GALNAC2", "PSMG2", "RAB31", "BABAM1", "TIMM44", "SLC25A24", 
           "SLC25A24", "SLC25A24", "STX6", "ARL8A", "C1orf31", "TRNAU1AP", 
           "PPIE", "PROCR", "STAU1", "PTPN1", "ADNP", "SLC37A1", "RPL3", 
           "ADSL", "NHP2L1", "CUL3", "CUL3", "GSK3B", "GSK3B", "GSK3B", 
           "GSK3B", "CDV3", "KIF9", "NDUFAF3", "HTT", "AFF4", "AFF4", "SKP1", 
           "UBE2D2", "DIMT1", "SCAF8", "MRPL18", "CDYL", "AKIRIN2", "AKIRIN2", 
           "YWHAZ", "PLEC", "SNX16", "SNX16", "RAD23B", "RAD23B", "CAMSAP1", 
           "RAP2C", "RAP2C", "GDI1")
df_GSK3B <- unique(df_GSK3B)
sets_list <- list(DEGs = degs, GSK3B = df_GSK3B)
ggVennDiagram(sets_list)
common_genes_GSK3B <- Reduce(intersect, sets_list)
common_genes_GSK3B

```
```{r}
df_CLOCK <- c("ADAMTS6", "AGR2", "ANP32E", "ARFGEF1", "ARL14", "ASNSD1", "BAZ1B", 
           "C1orf21", "C20orf72", "C2orf56", "C5orf13", "C9orf5", "CASK", 
           "CASP8", "CAST", "CAST", "CCDC111", "CCDC99", "CEBPG", "CEBPG", 
           "CETN3", "CFL2", "CHM", "CKAP2", "CLOCK", "CLOCK", "CLOCK", 
           "COMMD2", "CREB3L2", "CRYAB", "CSNK2A1", "CYCS", "CYP51A1", 
           "DAB2", "DCK", "DDX21", "DENND4A", "DIS3", "DPH5", "ECHDC1", 
           "ECT2", "EIF2S1", "EIF4E2", "EIF4G2", "EPS15", "FAM204A", 
           "FCHSD2", "FERMT2", "FGFRL1", "FIP1L1", "FMN1", "FMN2", "G2E3", 
           "GABPA", "GANC", "GBP1", "GNPDA2", "GNPNAT1", "GOLIM4", "GRAMD3", 
           "GULP1", "GXYLT1", "HAT1", "HAT1", "HBS1L", "HERC4", "HERC4", 
           "HIBCH", "HMG20B", "HMGB1P5", "HMGB3P6", "HNRNPA2B1", "HNRNPH3", 
           "HOXC5", "HOXC9", "HSPB6", "HSPH1", "HYLS1", "IKZF2", "IL13RA1", 
           "IL13RA2", "IL13RA2", "IPO8", "JRKL", "KIAA1324L", "KIF18A", 
           "KRR1", "LANCL1", "LARP1B", "LINC00327", "LMAN1", "LMO7", "LPP", 
           "LRP11", "MAGOHB", "MCM8", "MCTS1", "MFN1", "MGA", "MNAT1", 
           "MPHOSPH6", "MPP5", "MPP5", "MPP6", "MRPS10", "MTFR1", "MYO1B", 
           "NDUFB3", "NFYB", "NKIRAS1", "NOL10", "NRD1", "PDCD10", "PDCL3", 
           "PDCL3", "PDGFC", "PEG10", "PHACTR2", "PITX2", "PKN2", "PLCE1", 
           "PPID", "PPIL3", "PPIL3", "PPM1L", "PPP1R12A", "PRPF39", "PUS10", 
           "PWP1", "PWP1", "RAB10", "RAB23", "RAB3GAP2", "RABL3", "RAD21", 
           "RAD21", "RPAP3", "RPF1", "RPS6", "RUFY2", "SASH1", "SAV1", 
           "SEMA3A", "SERPINB7", "SET", "SETDB1", "SGCD", "SIPA1L1", "SKA2", 
           "SKA2", "SLC22A15", "SMARCE1", "SMU1", "SOCS4", "SOCS6", 
           "SPATS2L", "SRP9", "STX2", "SULF1", "SUV420H1", "SWAP70", "TAF13", 
           "TAF1A", "TBPL1", "TCEANC2", "TJP1", "TMEM87A", "TMOD3", "TPMT", 
           "TTC21B", "TTC39B", "TYW3", "UFSP2", "USP16", "UXS1", "VPS35", 
           "VTA1", "WDR33", "WDR44", "YAP1", "YWHAZ", "YWHAZ", "ZBTB8OS", 
           "ZFP106", "ZNF148", "ZNF569", "ZNF673", "ZNF765", "ZNF766", 
           "ZNF876P")
df_CLOCK <- unique(df_CLOCK)
sets_list <- list(DEGs = degs, CLOCK = df_CLOCK)
ggVennDiagram(sets_list)
common_genes_CLOCK <- Reduce(intersect, sets_list)
common_genes_CLOCK

```
```{r}
df_FYN <- c("AAK1", "ACAD11", "ACPP", "ADAMTS10", "ALS2", "ANKH", "APOL6", 
           "ARHGAP9", "BTBD9", "C21orf49", "CCDC109B", "CELF2", "CMC1", 
           "CNOT4", "DDB2", "EPC1", "ERCC5", "FAM96B", "FAM96B", "FYN", 
           "FYN", "FYN", "GPR35", "GRAMD3", "HERC1", "HERPUD2", "HS6ST1", 
           "IPCEF1", "ITPKB", "JMY", "KDM4C", "KLF12", "KLF12", "KLF12", 
           "KLF12", "KLF13", "KLF13", "KLF13", "LAX1", "LPAR5", "MAML2", 
           "MAPRE2", "MNT", "MORF4L1", "MUTYH", "NDNL2", "NKRF", "NTRK1", 
           "ODF2L", "PCGF5", "PDHX", "PITPNC1", "PNKD", "POU6F1", "PRKCH", 
           "PSME1", "PTAR1", "RAB2B", "RASA3", "RASA3", "RNF216", "RPGR", 
           "RTTN", "SCARNA17", "SEPT9", "SEPW1", "SIGIRR", "SLC25A5", 
           "SPR", "TMEM91", "TMEM91", "TNIK", "TRANK1", "TRERF1", "TRIB2", 
           "TRMT112", "USF2", "USP6NL", "ZBTB40", "ZNF264", "ZNF569", 
           "ZNF571", "ZNF836")
df_FYN <- unique(df_FYN)
sets_list <- list(DEGs = degs, FYN = df_FYN)
ggVennDiagram(sets_list)
common_genes_FYN <- Reduce(intersect, sets_list)
common_genes_FYN

```
```{r}
df_APP <- c("AKAP12", "AP2B1", "APBB2", "APLP2", "APLP2", "APP", "APP", 
           "APP", "APP", "APP", "APP", "APP", "APP", "APP", "APP", 
           "APP", "APP", "APP", "APP", "ATRN", "BCAP29", "BHLHB9", 
           "BMPR2", "BTBD10", "C6orf138", "C9orf30", "CACNB3", "CAMSAP2", 
           "CD97", "CDH13", "CDH13", "CDH13", "CLCN3", "CLSTN1", "CTNND1", 
           "CTTNBP2NL", "DDAH1", "DKK3", "DKK3", "DOCK1", "DYNC1H1", "EFNA5", 
           "EPHX4", "ERLIN1", "ERO1L", "EXD2", "FAM110A", "FBXL2", "FLRT2", 
           "FLRT2", "FLRT2", "FLRT2", "FOLR3", "FZD6", "GLE1", "HDAC9", "IQCA1", 
           "ITGA6", "ITGB1", "KIAA1704", "LAMC2", "LUZP1", "LUZP1", "LUZP1", 
           "MAST4", "MDM2", "NFIB", "NLGN4Y", "NTN4", "NUAK1", "NUAK1", "NUAK1", 
           "OCIAD1", "PAFAH1B2", "PDIA3", "PHACTR1", "PLOD2", "PON2", "PPFIBP1", 
           "PRRG1", "PRRG1", "PSAP", "PSORS1C1", "PTGFRN", "PTPRK", "RAB11A", 
           "RAB5B", "RAI14", "RASSF1", "RBFOX2", "RBMS2P1", "RIN2", "SHROOM4", 
           "SIAE", "SLC7A11", "SMAD1", "SMAD1", "SMARCD2", "SPTAN1", "SYNPO", 
           "THSD1", "TJP2", "TM4SF1", "TMCC1", "TMEM17", "TMEM30A", "TMTC3", 
           "TNFAIP1", "TRIM44", "TSPAN3", "TSPAN3", "UCP2", "UCP2", "VEZT", 
           "WRB", "YWHAE", "ZNF677")
df_APP <- unique(df_APP)
sets_list <- list(DEGs = degs, APP = df_APP)
ggVennDiagram(sets_list)
common_genes_APP <- Reduce(intersect, sets_list)
common_genes_APP

```


```{r}
df_SORL1 <- c("ACACB", "ACADVL", "ADK", "ARL6", "ATP8A1", "ATP8A1", "C20orf177", 
           "CABIN1", "CAMK2G", "CAMKMT", "CAMTA1", "CCDC84", "CDK13", "CYFIP2", 
           "CYFIP2", "DHRS7", "DIP2B", "DNAJB14", "DSTYK", "DUSP16", "DYRK1A", 
           "ENTPD4", "FAM134B", "FAM169A", "FBXO21", "FXC1", "GUSBP11", "HDAC1", 
           "HERPUD2", "IKBKAP", "INPP4A", "KIAA1468", "KIAA2026", "LMBR1", "MAPK8IP3", 
           "MFAP3L", "MLLT6", "MPP7", "PDK3", "PIP4K2A", "PLCL2", "PPM1B", "PPP1R3E", 
           "PRKCB", "RALGAPB", "RBL2", "RBM33", "RNF44", "ROGDI", "RPS6KA5", "RPS6KA5", 
           "SCYL2", "SECISBP2", "SEMA4D", "SENP6", "SFI1", "SGMS1", "SLAIN1", "SORL1", 
           "SORL1", "SORL1", "SORL1", "SORL1", "SORL1", "SORL1", "SPOCK2", "SPOCK2", 
           "SUN2", "TMEM66", "TPP2", "TTBK2", "TTC19", "TXLNA", "UBXN4", "VAMP1", 
           "ZNF815")
df_SORL1 <- unique(df_SORL1)
sets_list <- list(DEGs = degs, SORL1 = df_SORL1)
ggVennDiagram(sets_list)
common_genes_SORL1 <- Reduce(intersect, sets_list)
common_genes_SORL1

```
```{r}
common_genes_all <- Reduce(intersect, list(common_genes_GRIN2B, common_genes_KCNMA1, 
                                            common_genes_PLBC1, common_genes_NRTK2, 
                                            common_genes_APBB2, common_genes_GSK3B, 
                                            common_genes_CLOCK, common_genes_FYN, 
                                            common_genes_APP, common_genes_SORL1))
common_genes_all

 
```







```{r}
df_ALS <-read_excel("C:/Users/sanet/OneDrive/Documenti/Trento/UNI/Data Mining - Progetto/geneTabReportALS.xlsx")
ALSGenes <- df_ALS$`Mouse Homologs`
sets_list <- list(DEGs = DEGs, AlSGenes = ALSGenes)
ggVennDiagram(sets_list) 
common_genes2 <- Reduce(intersect, sets_list)
common_genes2

```



```{r}
df_SCH <-read_excel("C:/Users/sanet/OneDrive/Documenti/Trento/UNI/Data Mining - Progetto/geneTabReportSCZ.xlsx")
SCHGenes <- df_SCH$`Mouse Homologs`
sets_list <- list(DEGs = DEGs, SCHGenes = SCHGenes)
ggVennDiagram(sets_list) 
common_genes3 <- Reduce(intersect, sets_list)
common_genes3
```
```{r}
sets_list <- list(c1 = common_genes1, c2 = common_genes2 , c3 = common_genes3)
ggVennDiagram(sets_list) 
common_of_common <-Reduce(intersect, sets_list)
common_of_common
```



```{r}
library(devtools)
install_gitlab("medbio/disgenet2r")
library(disgenet2r)
api_key <- "1297474c-2bdb-4be1-838c-ec5a2c2af835"
Sys.setenv(DISGENET_API_KEY= api_key)
#results <- gene2evidence( gene = "GSK3B", vocabulary = "HGNC", database = "CURATED")
#tab <- results@qresult
markersup <- read.csv("C:/Users/sanet/OneDrive/Documenti/Trento/UNI/Data Mining - Progetto/PDEG.csv")

markers_sorted <- subset(markersup, is.finite(markersup$avg_log2FC))
markers_sorted <- markersup[order(-abs(markersup$avg_log2FC), markersup$p_val_adj), ]
top_10_DEGs <-head(markers_sorted$X ,20)
top_10_list_upper <- lapply(top_10_DEGs, function(x) {
  if (is.character(x)) {
    return(toupper(x))  # Convert character elements to uppercase
  } else {
    return(x)  # Keep non-character elements unchanged
  }
})
myListOfGenes<- as.list(top_10_list_upper)
results <- gene2disease(
  gene     = myListOfGenes,
  database = "CURATED",
  score =c(0.5, 1),
  verbose  = TRUE
)
tab <- unique(results@qresult[  ,c("gene_symbol", "disease_name","score", "yearInitial", "yearFinal")] )
plot( results,
      type = "Network",
      prop = 10, verbose = T)
plot( results, type="Network",
      class="DiseaseClass", nchars=60, interactive =F)

plot( results, type="Heatmap",
      class="DiseaseClass", nchars=60, interactive =F)
```

```{r}
library(devtools)
install_gitlab("medbio/disgenet2r")
library(disgenet2r)
api_key <- "1297474c-2bdb-4be1-838c-ec5a2c2af835"
Sys.setenv(DISGENET_API_KEY= api_key)
#results <- gene2evidence( gene = "GSK3B", vocabulary = "HGNC", database = "CURATED")
#tab <- results@qresult
markersup <- subset(markers, markers$avg_log2FC > 0)
markers_sorted <- subset(markersup, is.finite(markersup$avg_log2FC))
markers_sorted <- markersup[order(-abs(markersup$avg_log2FC), markersup$p_val_adj), ]
top_10_DEGs <-head(markers_sorted$X ,20)
top_10_list_upper <- lapply(top_10_DEGs, function(x) {
  if (is.character(x)) {
    return(toupper(x))  # Convert character elements to uppercase
  } else {
    return(x)  # Keep non-character elements unchanged
  }
})
myListOfGenes<- as.list(top_10_list_upper)
results <- gene2disease(
  gene     = myListOfGenes,
  database = "CURATED",
  score =c(0.5, 1),
  verbose  = TRUE
)
tab <- unique(results@qresult[  ,c("gene_symbol", "disease_name","score", "yearInitial", "yearFinal")] )
plot( results,
      type = "Network",
      prop = 10, verbose = T)
plot( results, type="Network",
      class="DiseaseClass", nchars=60, interactive =F)

plot( results, type="Heatmap",
      class="DiseaseClass", nchars=60, interactive =F)
```

